LiteCore/tests/CMakeLists.txt

```diff
@@ -92,6 +92,10 @@ add_executable(
     ${TOP}LiteCore/Support/TestsCommon.cc
     main.cpp
 )
+
+get_directory_property(this_targets DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR} BUILDSYSTEM_TARGETS)
+set(LITECORE_TARGETS ${LITECORE_TARGETS} ${this_targets} PARENT_SCOPE)
+set(LITECORE_TARGETS ${LITECORE_TARGETS} ${this_targets})
```

Comment:
This line doesn't appear to be necessary since LITECORE_TARGETS is not used anywhere in this file